src/main/java/net/minecraftforge/client/model/generators/BlockStateProvider.java

```diff
@@ -442,7 +464,7 @@ public void wallBlock(WallBlock block, ModelFile post, ModelFile side, ModelFile
         		wallSidePart(builder, sideTall, e, WallSide.TALL);
         	});
     }
-    
+
```

Comment:
Could you remove this whitespace change?

Reply:
Done